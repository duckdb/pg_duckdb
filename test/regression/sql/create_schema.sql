CREATE ROLE test_role SUPERUSER;
CREATE SCHEMA test_role;
DROP SCHEMA test_role;
CREATE SCHEMA AUTHORIZATION test_role;
DROP SCHEMA test_role;
DROP ROLE test_role;

CREATE ROLE ddb$_test_role SUPERUSER;
CREATE SCHEMA ddb$_test_role; -- error
CREATE SCHEMA AUTHORIZATION ddb$_test_role; -- error
