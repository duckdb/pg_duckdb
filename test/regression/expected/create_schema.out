CREATE ROLE test_role SUPERUSER;
CREATE SCHEMA test_role;
DROP SCHEMA test_role;
CREATE SCHEMA AUTHORIZATION test_role;
DROP SCHEMA test_role;
DROP ROLE test_role;
CREATE ROLE ddb$_test_role SUPERUSER;
CREATE SCHEMA ddb$_test_role; -- error
ERROR:  Creating ddb$ schemas is currently not supported
CREATE SCHEMA AUTHORIZATION ddb$_test_role; -- error
ERROR:  Creating ddb$ schemas is currently not supported
